python/lsst/faro/base/ConfigBase.py

```diff
@@ -0,0 +1,93 @@
+# This file is part of faro.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from lsst.pex.config import Config, Field
+from lsst.pex.config.callStack import getStackFrame
+from lsst.pipe.tasks.configurableActions import ConfigurableActionStructField
+
+
+class ColumnField(Field):
+    """A configuration field (`lsst.pex.config.Field` subclass) that indicates
+    a band-independent column name."""
+
```

Comment:
We should add docstrings here to explain what all the keyword params are for.

Reply:
@bechtol  could you do this?